src/timeMachine/components/view_options/GeoOptions.tsx

```diff
@@ -0,0 +1,126 @@
+import React, {FC} from 'react'
+import {useDispatch} from 'react-redux'
+import CustomHeatMapOptions from 'src/timeMachine/components/view_options/CustomHeatmapOptions'
+import {
+  setLatitude,
+  setLongitude,
+  setAllowPanAndZoom,
+  setZoomValue,
+  setMapType,
+} from 'src/timeMachine/actions'
+import {GeoViewProperties} from 'src/types'
+import {Grid, SelectGroup, Form, RangeSlider} from '@influxdata/clockface'
+
+const selectOptions = ['Point', 'Circle', 'Heat', 'Track']
+const mapTypeHash = {
+  Point: 'pointMap',
+  Circle: 'circleMap',
+  Heat: 'heatmap',
+  Track: 'trackMap',
+}
+
+const displayCustomOptions = (mapType: string) => {
+  /* Eventually, the other map type subtype customizations can be added here */
+
+  switch (mapType) {
+    case 'heatmap':
+      return <CustomHeatMapOptions />
+    case 'trackMap':
+    case 'pointMap':
+    case 'circleMap':
+    default:
+      return null
+  }
+}
+
+const GeoOptions: FC<GeoViewProperties> = props => {
+  const dispatch = useDispatch()
+  const handleSelectGroupClick = (activeOption: string): void => {
+    dispatch(setMapType(mapTypeHash[activeOption]))
+  }
+  const handleLatLongSelect = (latLonVal: number, type: string = 'lat') => {
```

Comment:
not a huge fan of this type of function design. we have to pass in a value that represents either a latitude or a longitude depending on the second argument that sometimes gets passed in.

much clearer illustration of intent (and future use) to do something like this:

```ts
handleSelectLatitude(latitude: number) =>
  dispatch(setLatitude(latitude))

handleSelectLongitude(longitude: number) =>
  dispatch(SetLongitude(longitude))
```

then when it's called, you can just call 
```ts
  onChange={event => selectLatitude(parseFloat(event.target.value))}
```

this way, in the future, if there are some fancy business rules around how we select latitude, like say we have to apply a projection to it, we can easily do that without changing any code that consumes this function.